app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntryEffectHandler.kt

```diff
@@ -58,4 +67,20 @@ object BloodPressureEntryEffectHandler {
           .map(::BloodPressureMeasurementFetched)
     }
   }
+
+  private fun showBpValidationError(
+      ui: BloodPressureEntryUi,
+      validationError: ShowBpValidationError
+  ) {
+    when (validationError.result) {
+      is ErrorSystolicLessThanDiastolic -> ui.showSystolicLessThanDiastolicError()
+      is ErrorSystolicTooHigh -> ui.showSystolicHighError()
+      is ErrorSystolicTooLow -> ui.showSystolicLowError()
+      is ErrorDiastolicTooHigh -> ui.showDiastolicHighError()
+      is ErrorDiastolicTooLow -> ui.showDiastolicLowError()
+      is ErrorSystolicEmpty -> ui.showSystolicEmptyError()
+      is ErrorDiastolicEmpty -> ui.showDiastolicEmptyError()
+      is Success -> { /* Nothing to do here. */ }
+    }
```

Comment:
Should we add a `.exhaustive()` call here?

Reply:
Fixed.